backend/src/controllers/v2/secretsController.ts

```diff
@@ -1378,3 +1378,18 @@ export const deleteSecrets = async (req: Request, res: Response) => {
     secrets: req.secrets
   });
 };
+
+
+
+export const moveSecretsToFolder = async (req: Request, res: Response) => {
+  const { folderId } = req.params;
+  const { secretIds } = req.body;
+
+  const updateSecretsFolder = await Secret.updateMany(
```

Comment:
The folder id will need additional check like

1. Folder exist
2. Given folder id is in same environment and workspace. 
3. Limit this operation to a set of environment, workspaceid and folder

Reply:
Alright would look into these

Reply:
Addressed!

Reply:
@EBEN4REAL Kindly check folder exist. Folder model contains all the folders inside of an env.

Reply:
I have done that. I pushed the changes already